articles/openshift/howto-using-aro.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Using `az aro` | Microsoft Docs
+description: Create, access, and delete clusters with `az aro` extension
+author: klamenzo 
+ms.author: b-lejaku
+ms.service: container-service
+ms.topic: conceptual
+ms.date: 03/02/2020
+keywords: aro, openshift, az aro, red hat, cli
+#Customer intent: As a customer, I want to create an ARO custer using the command line.
+---
+
+# Using `az aro`
+
+The `az aro` extension allows you to create, access, and delete Azure Red Hat OpenShift clusters directly from the command line using the Azure CLI.
```

Comment:
@sabbour I removed the following line: 
"If you have a whitelisted subscription, it can be used against the pre-GA Azure Red Hat OpenShift v4 service, or (by setting RP_MODE=development) it can be used against a development RP running at https://localhost:8443/."

Should we keep something about "using pre-GA Azure Red Hat OpenShift v4 service", or something about this being in preview?

Reply:
the fact that it's preview needs to be very clear.